Gemfile

```diff
@@ -1,26 +1,16 @@
 source 'https://rubygems.org'
 
-gemspec
-gem 'hashie'
-gem 'json'
-gem 'kitchen-sync'
-gem 'test-kitchen'
-
 group :vagrant do
   gem 'vagrant-wrapper'
   gem 'kitchen-vagrant'
 end
 
-group :windows do
```

Comment:
you can just leave this in here.  It is needed for running against windows instances from a linux test box too, and should be installed if you are testing against windows.